src/backends/utils.js

```diff
@@ -12,22 +10,25 @@ const log = require('../common').log('fixerio')
  * @param {String} destination The URI of the destination ledger
  * @return {Array}
  */
-function getCurrencyPair (sourceLedger, destinationLedger) {
-  if (!_.isEmpty(config.get('tradingPairs'))) {
-    for (let pair of config.get('tradingPairs')) {
-      // trading pair is of the form
-      // ["<currency@<source_ledger>","<currency>@<destination_ledger>"]
-      if (pair[0].indexOf(sourceLedger) === 4 &&
-        pair[1].indexOf(destinationLedger) === 4) {
-        return [pair[0].slice(0, 3), pair[1].slice(0, 3)]
+function getCurrencyPairFromConfig (config) {
+  const log = require('../common').log(config.get('backend'))
+  return function getCurrencyPair (sourceLedger, destinationLedger) {
```

Comment:
Could you just use partial function application instead of a function that returns a function?